### ISSUE TITLE
"The Animals!" does not boot

### ISSUE THREAD

**Issue by Beatatester** [MiSTer-devel/MegaCD_MiSTer#39]
"The Animals!" does not boot. If you attempt to play the game on MiSTer, the standard Sega CD intro screen displays (sonic and the sega logo, and the text "produced by or under license from sega enterprises ltd") then the screen goes black and the game never loads.

I have tested both Redump verisons of the game ("Animals!, The (USA)" and "Animals!, The (Europe)") and both versions show the same behavior. 

I have tested the USA version on real hardware and on the Mega SD, where it plays normally.

**Comment by SegaSnatcher** [comment]
Have you tried different Bios?

**Comment by Beatatester** [comment]
Yes. I've tried multiple BIOS and gotten the same result. The USA BIOS I've tried include the following:

[BIOS] Sega CD (USA) (v1.00)
[BIOS] Sega CD (USA) (v1.10)
[BIOS] Sega CD 2 (USA) (v2.00)
[BIOS] Sega CD 2 (USA) (v2.00W)
[BIOS] Sega CD 2 (USA) (v2.11X)
[BIOS] CDX (USA) (v2.21X)

I've also tried several of the European BIOS files for the European release. Is there a specific BIOS that is known to help with this game?
